source/cluster-man/doveadm-cluster-backend.1.rst.in

```diff
@@ -55,6 +55,9 @@ cluster backend add
 
 Adds a new backend to cluster. New backends are added in standby state. UUID should not be used unless you are re-adding the same backend.
 
+*name*
+  must be a resolvable DNS name. Unresolvable DNS names and IPv4/v6 literal addresses will be rejected.
```

Comment:
Completely unrelated, but I saw that `source/man/doveadm-process-status.1.rst` is using a completely different format to the argument itemization. Might make sense to amend this PR with a cleanup-commit?

Reply:
Do you mean having an 'Arguments' section ?
I can do that, but, it seems a bit dispersive to have the note about what can be used as 'name' there instead that under the only command using it.
If it isn't that, i think i don't understand what you are trying to address.